cmd/windup.go

```diff
@@ -90,12 +130,6 @@ func (r *Windup) options() (options command.Options, err error) {
 	if err != nil {
 		return
 	}
-	if r.Rules != nil {
-		err = r.Rules.AddOptions(&options)
-		if err != nil {
-			return
-		}
-	}
```

Comment:
**hint**: This just got moved to be executed ahead of processing the list of targets passed in on the task.Data.